cli/clistat/stat.go

```diff
@@ -17,15 +15,65 @@ import (
 	sysinfotypes "github.com/elastic/go-sysinfo/types"
 )
 
+// Prefix is a scale multiplier for a result.
+// Used when creating a human-readable representation.
+type Prefix float64
+
+const (
+	PrefixDefault = 1.0
+	PrefixKibi    = 1024.0
+	PrefixMebi    = PrefixKibi * 1024.0
+	PrefixGibi    = PrefixMebi * 1024.0
+	PrefixTebi    = PrefixGibi * 1024.0
```

Comment:
```suggestion
	PrefixMibi    = PrefixKibi * 1024.0
	PrefixGibi    = PrefixMebi * 1024.0
	PrefixTibi    = PrefixGibi * 1024.0
```

Reply:
to be consistent with line 32 and 34

Reply:
These are the canonical prefix names according to NIST: https://physics.nist.gov/cuu/Units/binary.html